### PR TITLE
added gesture test

### PR DESCRIPTION
**Pull request by saikrishna321**
* LongPressTest

* swipeTest

* DoubleTapTest

### CODE REVIEW

File: sample-code/examples/java/junit/src/test/java/com/saucelabs/appium/GestureTest.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+import org.openqa.selenium.Dimension;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import static org.junit.Assert.assertTrue;
+
+public class GestureTest extends BaseCrossPlatformDriver{
+
+
+    public void horizontalSwipingTest() throws Exception {
```
Comment (SrinivasanTarget) [inline sample-code/examples/java/junit/src/test/java/com/saucelabs/appium/GestureTest.java]: @Test here

File: sample-code/examples/java/junit/src/test/java/com/saucelabs/appium/GestureTest.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        MobileElement slider = driver.findElementByAccessibilityId("slider");
+        Dimension size = slider.getSize();
+
+        TouchAction swipe = new TouchAction(driver).press(slider, 0, size.height / 2).waitAction(2000)
+                .moveTo(slider, size.width / 2, size.height / 2).release();
+        swipe.perform();
+    }
+
+    //Will work only on XCUI mode as doubleTap is not supported by IOS/Android
+    public void doubleTap() throws InterruptedException {
```
Comment (SrinivasanTarget) [inline sample-code/examples/java/junit/src/test/java/com/saucelabs/appium/GestureTest.java]: @Test here too

File: sample-code/examples/java/junit/src/test/java/com/saucelabs/appium/BaseCrossPlatformDriver.java [hunk 3]
```diff
@@ -72,4 +75,14 @@ public void login() {
         new WebDriverWait(driver, 30).until(ExpectedConditions.
                 elementToBeClickable(MobileBy.AccessibilityId("login"))).click();
     }
+
+    public void verticalSwipe(String locator) throws InterruptedException {
```
Comment (SrinivasanTarget) [inline sample-code/examples/java/junit/src/test/java/com/saucelabs/appium/BaseCrossPlatformDriver.java]: Can we add this also under GestureTest class?

File: sample-code/examples/java/junit/src/test/java/com/saucelabs/appium/GestureTest.java (hunk 1 above)
Comment (saikrishna321) [inline sample-code/examples/java/junit/src/test/java/com/saucelabs/appium/GestureTest.java]: Done

File: sample-code/examples/java/junit/src/test/java/com/saucelabs/appium/GestureTest.java (hunk 2 above)
Comment (saikrishna321) [inline sample-code/examples/java/junit/src/test/java/com/saucelabs/appium/GestureTest.java]: Done

File: sample-code/examples/java/junit/src/test/java/com/saucelabs/appium/BaseCrossPlatformDriver.java (hunk 3 above)
Comment (saikrishna321) [inline sample-code/examples/java/junit/src/test/java/com/saucelabs/appium/BaseCrossPlatformDriver.java]: For this the application needs a fix, will push updated app and test in the next PR

**Pull request merged**